types/heremaps/index.d.ts

```diff
@@ -3112,7 +3175,7 @@ declare namespace H {
                 /**
                  * The invalidation mark represents a counter which is increased whenever an invalidation takes place.
                  */
-                export interface Mark { }
+                type Mark = any;
```

Comment:
I tried this but `type Mark = {}` leads to
```ERROR: 3178:22  interface-over-type-literal  Use an interface instead of a type literal.```
And keeping the previous `interface Mark {}` throws
```ERROR: 3178:27  no-empty-interface  An empty interface is equivalent to `{}`.```
So how to solve it properly?